core/src/utilities/helpers/generic-helpers.js

```diff
@@ -209,26 +209,23 @@ class GenericHelpersClass {
     return processedString;
   }
 
-  getInnerHeight() /* istanbul ignore next */ {
+  getInnerHeight() {
     return LuigiElements.isCustomLuigiContainer() ? LuigiElements.getLuigiContainer().clientHeight : window.innerHeight;
   }
 
-  getContentAreaHeight /* istanbul ignore next */() {
```

Comment:
Why did you remove the comments?

Reply:
I think i was something from auto format file but I revers all these changes

Reply:
It belongs to our code coverage.